lib/rdf/changeset.rb

```diff
@@ -156,6 +156,12 @@ def delete_statement(statement)
       self.deletes << statement
     end
 
+    # This simply returns its argument as a query in order to trick
+    # {RDF::Mutable#delete} into working.
+    def query stmt
```

Comment:
Style in RDF.rb is to use parens for method definitions.

```suggestion
    def query(stmt)
```